modules/combined/tests/thermal_elastic/thermal_elastic.i

```diff
@@ -83,133 +85,36 @@
     x = '0     1     2'
     y = '100.0 100.0 500.0'
   [../]
-  [./ym_func]
-    type = PiecewiseLinear
-    x = '100 500'
-    y = '1e6 6e5'
-  [../]
-  [./pr_func]
-    type = PiecewiseLinear
-    x = '100 500'
-    y = '0   0.25'
-  [../]
-
-[] # Functions
+[]
 
 [Variables]
 
-  [./disp_x]
-    order = FIRST
-    family = LAGRANGE
-  [../]
-
-  [./disp_y]
-    order = FIRST
-    family = LAGRANGE
-  [../]
-
-  [./disp_z]
-    order = FIRST
-    family = LAGRANGE
-  [../]
-
   [./temp]
     order = FIRST
     family = LAGRANGE
     initial_condition = 100.0
   [../]
 
-[] # Variables
-
-[AuxVariables]
-
-  [./stress_xx]
-    order = CONSTANT
-    family = MONOMIAL
-  [../]
-  [./stress_yy]
-    order = CONSTANT
-    family = MONOMIAL
-  [../]
-  [./stress_zz]
-    order = CONSTANT
-    family = MONOMIAL
-  [../]
-  [./stress_xy]
-    order = CONSTANT
-    family = MONOMIAL
-  [../]
-  [./stress_yz]
-    order = CONSTANT
-    family = MONOMIAL
-  [../]
-  [./stress_zx]
-    order = CONSTANT
-    family = MONOMIAL
-  [../]
-
-[] # AuxVariables
+[]
 
-[SolidMechanics]
-  [./solid]
-    disp_x = disp_x
-    disp_y = disp_y
-    disp_z = disp_z
+[Modules/TensorMechanics/Master]
+  [./all]
+    add_variables = true
+    generate_output = 'stress_xx stress_yy stress_zz stress_xy stress_xz stress_yz'
+    strain = FINITE
+    block = '1 2 3 4 5 6 7'
```

Comment:
if you omit the block parameter no restriction is applied (which is what the `all` name seems to indicate you want)